Dockerfile

```diff
@@ -24,25 +24,24 @@ RUN \
     lmdb-dev \
     flatbuffers-dev \
     libsecp256k1-dev \
-    zstd-dev \
-  && rm -rf /var/cache/apk/* \
```

Comment:
we already have `--no-cache` in the runtime build:

> docker images strfry_image_ubuntu

REPOSITORY            TAG       IMAGE ID       CREATED         SIZE
strfry_image_ubuntu   latest    312188f2bc47   3 minutes ago   206MB

REPOSITORY              TAG       IMAGE ID       CREATED          SIZE
strfry_image_original   latest    bba6a16b8521   14 seconds ago   143MB

REPOSITORY     TAG       IMAGE ID       CREATED         SIZE
strfry_image   latest    22a7b0451b35   7 minutes ago   143MB